etc/DependencyInstaller.sh

```diff
@@ -292,6 +292,65 @@ EOF
     _installHomebrewPackage "spdlog" "0974b8721f2f349ed4a47a403323237e46f95ca0"
 }
 
+_installDebianCleanUp() {
+    apt-get autoclean -y
+    apt-get autoremove -y
+}
+
+_installDebianDev() {
+    export DEBIAN_FRONTEND="noninteractive"
+    apt-get -y update
+    apt-get -y install tzdata
+    apt-get -y install \
+        automake \
+        autotools-dev \
+        build-essential \
+        bison \
+        flex \
+        clang \
+        g++ \
+        gcc \
+        git \
+        lcov \
+        libpcre2-dev \
+        libpcre3-dev \
+        python3-dev \
+        libreadline-dev \
+        tcl-dev \
+        tcllib \
+        wget \
+        zlib1g-dev \
+        libomp-dev
+}
+
+_installDebianRuntime() {
+    export DEBIAN_FRONTEND="noninteractive"
+    apt-get -y update
+    apt-get -y install tzdata
+    apt-get install -y \
+        binutils \
+        libgomp1 \
+        libtcl \
+        qt5-image-formats-plugins \
+        tcl-tclreadline
+    
+    if [[ $1 == 11 ]]; then
+        apt-get install -y \
+            libpython3.8 \
+            qtbase5-dev \
+            qtchooser \
+            qt5-qmake \
+            qtbase5-dev-tools
+    else
+        apt-get install -y \
+            libpython3.7 \
+            qt5-default
+    fi
+
+    # need the strip "hack" above to run on docker
+    strip --remove-section=.note.ABI-tag /usr/lib/x86_64-linux-gnu/libQt5Core.so
```

Comment:
Is this something we want to move to Dockerfile or does this has no effect on installs outside docker?

Reply:
I copied it from ubuntu's installation because Debian is so similar to it, however, I can try to push the code without it to notice its effect.